types/three/src/renderers/webxr/WebXRManager.d.ts

```diff
@@ -16,15 +26,24 @@ export class WebXRManager extends EventDispatcher {
      */
     isPresenting: boolean;
 
+    /**
+     * @default true
+     */
+    cameraAutoUpdate: boolean;
+
     getController(index: number): Group;
     getControllerGrip(index: number): Group;
     getHand(index: number): Group;
```

Comment:
These are not documented/typed, but they come from WebXRController's various space methods:
 - https://github.com/mrdoob/three.js/blob/dev/src/renderers/webxr/WebXRController.js#L16

I have left them as-is until I (or someone) documents them in three just in case anything looks wrong.